src/models/github-server-app.test.ts

```diff
@@ -128,6 +135,92 @@ describe("GitHubServerApp", () => {
 					}
 				});
 			});
+
+			describe("GitHubServerApp update", () => {
+				const gitHubBaseUrl = "http://myinternalinstance.com";
+				const installationId = 42;
+				const uuid = "c97806fc-c433-4ad5-b569-bf5191590be2";
+				const clientIdWithTypo = "lvl.1n23j12389wndd";
+				const originalWebhookSecret = "anothersecret";
+				const originalPrivateKey = "privatekey";
+				let gitHubServerApp;
+
+				beforeEach(async () => {
+					gitHubServerApp = await GitHubServerApp.create({
+						uuid,
+						appId: 1,
+						gitHubAppName: "my awesome app",
+						gitHubBaseUrl,
+						gitHubClientId: clientIdWithTypo,
+						gitHubClientSecret: "secret",
+						webhookSecret: "anothersecret",
+						privateKey: "privatekey",
+						installationId
+					});
+				});
+
+				it("should update GitHub app when uuid is found", async () => {
+					const originalApp = await GitHubServerApp.findForUuid(uuid);
+					expect(originalApp && originalApp.gitHubClientId).toEqual(clientIdWithTypo);
+					expect(originalApp && originalApp.webhookSecret).toEqual(`encrypted:${originalWebhookSecret}`);
+					expect(originalApp && originalApp.privateKey).toEqual(`encrypted:${originalPrivateKey}`);
+
+					await GitHubServerApp.updateGitHubAppByUUID({
+						uuid,
+						appId: 1,
+						gitHubAppName: "my awesome app",
+						gitHubBaseUrl,
+						gitHubClientId: "lvl.1n23j12389wnde",
```

Comment:
/nit
Use const variable to better show the intention of the test. It took me a while to figure out where is the "typo".
So maybe either use some const var, or simpler, set the string vaule "origin_client_id" vs "new_client_id", so that the test is more readable. And "origin_webhook_secret" vs "new_webhook_secret".